Formula/zsh-async.rb

```diff
@@ -0,0 +1,28 @@
+class ZshAsync < Formula
+  desc "Perform tasks asynchronously without external tools"
+  homepage "https://github.com/mafredri/zsh-async"
+  url "https://github.com/mafredri/zsh-async/archive/v1.8.5.tar.gz"
+  sha256 "3ba4cbc6f560bf941fe80bee45754317dcc444f5f6114a7ebd40ca04eb20910a"
+  license "MIT"
+
+  bottle :unneeded
+
+  def install
+    cp "async.zsh", "async"
+    zsh_function.install "async"
```

Comment:
```suggestion
    zsh_function.install "async.zsh" => "async"
```

Reply:
@SMillerDev That removes the `async.zsh` from the root of the folder, which makes `test.zsh` (in the `test do` block) fail. Also it doesn't seem to like it if it's a symlink either…